packages/next-upgrade/README.md

```diff
@@ -0,0 +1,31 @@
+# @next/upgrade
+
+Upgrade Next.js apps to newer or beta versions with one command.
+
+# Build
```

Comment:
nit: this is a second h1 (`#`) aside from the top one.

Also might be worth mentioning the consumer-oriented `npx` command near the top and then build instructions for contributors as a secondary thing below it in the README.

Reply:
Following a basic one-line of use, it may also be worthwhile having a `## Why?` section explaining why the package exists. i.e. explaining that updating Next isn't just merely `pnpm update next`, that there are React dependencies, etc.

Reply:
Updated now!